### PR TITLE
[REF] requirements.txt: Update packages version

### PR DESCRIPTION
**Pull request by moylop260**
Closes https://github.com/OCA/pylint-odoo/issues/281

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,16 +1,12 @@
...
-
-astroid==2.2.0 ;python_version >= '3'
-pylint==2.3.0 ;python_version >= '3'
-pylint-plugin-utils==0.5 ;python_version >= '3'
-
-docutils==0.14
-lxml>=4.2.3
-Pygments==2.2
-restructuredtext_lint==1.1.3
+docutils==0.16
```
Comment (pedrobaeza) [inline requirements.txt:1]: Not sure if this will collapse with MQT + ocabot version, as it's fixed in 0.15.1.

cc @sbidoul

Comment (sbidoul) [reply requirements.txt:1]: That should be ok, since it was pinned at 0.14 before which is also different from 0.15.1.

Comment (pedrobaeza) [reply requirements.txt:1]: Isn't the upper one used then (now 0.16)?

Comment (pedrobaeza) [reply requirements.txt:1]: OK, not sure what is installed in each part. I was just rising the possible problem. It seems you are OK anyway, as README generator is handled outside Travis scope.

**Pull request merged**